src/pudl/metadata/resources/eia860.py

```diff
@@ -476,6 +476,36 @@
         "sources": ["eia860", "eia923"],
         "etl_group": "eia860",
     },
+    #     "emissions_control_equipment_eia860": {
+    #         "description": (
+    #             """
+    # The cost, type, operating status, retirement date, and install year of emissions control
+    # equipment reported to EIA. Includes control ids for sulfur dioxide (SO2), particulate
+    # matter, mercury, nitrogen oxide (NOX), and acid (HCl) gas monitoring.
+    # """
```

Comment:
This whole section is commented out because it's not currently working (I think due to foreign key/harvesting stuff)